indico/modules/events/contributions/util.py

```diff
@@ -231,6 +231,8 @@ def contribution_type_row(contrib_type):
 @memoize_request
 def get_contributions_with_user_as_submitter(event, user):
     """Get a list of contributions in which the `user` has submission rights"""
+    if not user:
```

Comment:
why not change the code calling it? this function is not supposed to be called without a user